.github/workflows/install-testing.yml

```diff
@@ -0,0 +1,86 @@
+name: Installation Tests
+
+on:
+  push:
+    branches:
+      - trunk
+    # Always test the workflow after it's updated.
+    paths:
+      - '.github/workflows/install-testing.yml'
+  pull_request:
+    # Always test the workflow when changes are suggested.
+    paths:
+      - '.github/workflows/install-testing.yml'
+  workflow_dispatch:
+    inputs:
+      new-version:
+        description: 'The version to test installing.'
+        type: string
+        default: 'latest'
```

Comment:
I wanted to avoid making the version required, but also didn't want to have the latest version hard coded here. This allows the default to be whatever the most recent release is (WP CLI default behavior), any `x.y` or `x.y.z` version, or even `nightly`.